cernopendata/modules/fixtures/data/docs/delphi-guide-eventserver/delphi-guide-eventserver.md

```diff
@@ -0,0 +1,36 @@
+# Raw data access: the DELPHI event server
+
+Raw data reconstruction is possible for the years 1992 and later, for individual events, via the DELPHI event server, `des`.
+
+## Requirements
+Running the even server requires access to the full DELPHI database, as well as the DELPHI raw data samples. It can be run from a container with access to the EOS file system, or locally from a machine with access to `cvmfs` and `EOS`.
+
+## Running the DELPHI event server
+
+A quick overview over the options is available from the tool itself, by just running
+
+```
+$ des
```

Comment:
BTW if in the Docker guide you have used a convention where `$` is used for system prompt and `[delphi ~] $` was used to denote VM or Docker prompt, perhaps we could keep this convention for the other docs pages and use here the prompt `[delphi ~] $` too? Just thinking out loud...